---
---

# André Lanrezac  
Bioinformatician | Scientific Developer

<span class="iconify" data-icon="charm:person"></span> [https://lanzac.github.io](https://lanzac.github.io)
  : <span class="iconify" data-icon="tabler:brand-github"></span> [github.com/lanzac](https://github.com/lanzac)

<span class="iconify" data-icon="ic:outline-location-on"></span> Paris, France
  : <span class="iconify" data-icon="tabler:mail"></span> [a.lanrezac@gmail.com](mailto:a.lanrezac@gmail.com)

## Education

**Ph.D. in Bioinformatics** – Université Paris Cité
  : **2023**

Supervised by [Marc Baaden](https://www.baaden.ibpc.fr/)

**B.S. in Biology & M.S. in Bioinformatics** – Sorbonne Université, Paris
  : **2019**


## Experience

**Research Assistant & Reseach engineer**
  : **Nov 2019 - June 2024**

[Laboratory of Theoretical Biochemistry, IBPC · CNRS , Paris](https://www-lbt.ibpc.fr/)

- Lead author of a review article on interactive molecular simulations (IMS) \[[~1]\], summarizing decades of work by the scientific community
- Enhanced the molecular visualization software UnityMol (Unity, C#) and the molecular engine BioSpring (C++) for real-time experiments on protein insertion into an analytical membrane \[[~2], [~3], [~5]\]
- Developed UNILIPID, a method and specialized bioinformatic Python tool for adapting lipid–protein interaction parameters, enabling the extension of implicit membrane representations from all-atom to coarse-grained models, and allowing simulations with curved membrane geometries \[[~3]\]
- Participated in the maintenance and release process of BioSpring by building a CI/CD pipeline (GitHub Actions, Docker) to automate releases, resolve multi-year dependency issues, and enhance software accessibility


**Bioinformatics Drug Design Intern**
  : **Feb 2019 - June 2019**

[Bioinformatics Research Team, ISYEB · Sorbonne université , Paris](https://impmc.sorbonne-universite.fr/fr/equipes/biophysique_et_bioinformatique.html)

Supervised by [Dirk Stratmann](http://www-ext.impmc.upmc.fr/~stratmann/)

- Developed a workflow combining Autodock with REMD simulations in GROMACS to assess binding affinity and stability of cyclic peptides, comparing results with HADDOCK to optimize peptide designs

- Manually designed cyclic peptides using UCSF-Chimera to optimize sequences and structures for targeted protein pockets, ensuring high binding affinity and stability


**Bioinformatics Structural Analysis Intern**
  : **July 2018**

[Bioinformatics Research Team, ISYEB · Sorbonne université , Paris](https://isyeb.mnhn.fr/fr/atelier-de-bio-informatique-384)

Supervised by [Mathilde Carpentier](https://isyeb.mnhn.fr/fr/annuaire/mathilde-carpentier-368)

- Continued development of [YAKUSA](https://pubmed.ncbi.nlm.nih.gov/16049912/), a fast protein structural similarity search tool based on shared high-scoring structural pairs (SHSPs)

- Re-implemented Bellman's algorithm (Python) to compute the longest path through shared motifs and optimize similarity search

- Developed heuristic algorithms to accelerate structural alignment and improve matching speed


## Skills

**Programming:**  Python, C++, C#, Bash, C, HTML, CSS

**Softwares & Frameworks:** Django, React, Unity, CMake, SQL

**DevOps:** Git, CI/CD (GitHub Actions), Docker

**Languages:** French (native), English (proficient), Mandarin (beginner)


## Publications

[~1]: André Lanrezac, Nicolas Férey, and Marc Baaden. “Wielding the power of interactive molecular
simulations”. WIREs Computational Molecular Science (July 2022). DOI:[10.1002/wcms.1594](https://onlinelibrary.wiley.com/doi/10.1002/wcms.1594).

[~2]: André Lanrezac et al. “Fast and Interactive Positioning of Proteins within Membranes”. Algorithms
(Nov. 2022). DOI:[10.3390/a15110415](https://www.mdpi.com/1999-4893/15/11/415).

[~3]: André Lanrezac and Marc Baaden. “UNILIPID, a Methodology for Energetically Accurate Prediction
of Protein Insertion into Implicit Membranes of Arbitrary Shape”. Membranes (Mar. 2023).
DOI:[10.3390/membranes13030362](https://www.mdpi.com/2077-0375/13/3/362).

[~4]: André Lanrezac, Nicolas Férey, and Marc Baaden. “Interactive Molecular Dynamics”. Reference Module in Chemistry, Molecular Sciences and Chemical Engineering. Elsevier, 2023.
DOI:[10.1016/B978-0-12-821978-2.00115-X](https://linkinghub.elsevier.com/retrieve/pii/B978012821978200115X).

[~5]: Laurent B, Lanrezac A, Santuz H, Ferey N, Delalande O, Baaden M. “BioSpring: An elastic network framework for interactive exploration of macromolecular mechanics”. Protein Science (April 2025). DOI:[10.1002/pro.70130/](https://doi.org/10.1002/pro.70130)