<!DOCTYPE html PUBLIC "-//W3C//DTD HTML 4.0 Transitional//EN" "http://www.w3.org/TR/REC-html40/loose.dtd"> <html><body> <h1 id="andré-lanrezac">André Lanrezac</h1> <p>Bioinformatician | Scientific Developer</p> <dl> <dt> <span class="iconify" data-icon="charm:person"></span> <a href="https://lanzac.github.io">https://lanzac.github.io</a> </dt> <dd> <span class="iconify" data-icon="tabler:brand-github"></span> <a href="https://github.com/lanzac" rel="external nofollow noopener" target="_blank">github.com/lanzac</a> </dd> <dt> <span class="iconify" data-icon="ic:outline-location-on"></span> Paris, France</dt> <dd> <span class="iconify" data-icon="tabler:mail"></span> <a href="mailto:a.lanrezac@gmail.com">a.lanrezac@gmail.com</a> </dd> </dl> <h2 id="education">Education</h2> <dl> <dt> <strong>Ph.D. in Bioinformatics</strong> – Université Paris Cité</dt> <dd><strong>2023</strong></dd> </dl> <p>Supervised by <a href="https://www.baaden.ibpc.fr/" rel="external nofollow noopener" target="_blank">Marc Baaden</a></p> <dl> <dt> <strong>B.S. in Biology &amp; M.S. in Bioinformatics</strong> – Sorbonne Université, Paris</dt> <dd><strong>2019</strong></dd> </dl> <h2 id="experience">Experience</h2> <dl> <dt><strong>Research Assistant &amp; Reseach engineer</strong></dt> <dd><strong>Nov 2019 - June 2024</strong></dd> </dl> <p><a href="https://www-lbt.ibpc.fr/" rel="external nofollow noopener" target="_blank">Laboratory of Theoretical Biochemistry, IBPC · CNRS , Paris</a></p> <ul> <li>Lead author of a review article on interactive molecular simulations (IMS) [<a href="Andr%C3%A9%20Lanrezac,%20Nicolas%20F%C3%A9rey,%20and%20Marc%20Baaden.%20%E2%80%9CWielding%20the%20power%20of%20interactive%20molecular">~1</a>], summarizing decades of work by the scientific community</li> <li>Enhanced the molecular visualization software UnityMol (Unity, C#) and the molecular engine BioSpring (C++) for real-time experiments on protein insertion into an analytical membrane [<a href="Andr%C3%A9%20Lanrezac%20et%20al.%20%E2%80%9CFast%20and%20Interactive%20Positioning%20of%20Proteins%20within%20Membranes%E2%80%9D.%20Algorithms">~2</a>, <a href="Andr%C3%A9%20Lanrezac%20and%20Marc%20Baaden.%20%E2%80%9CUNILIPID,%20a%20Methodology%20for%20Energetically%20Accurate%20Prediction">~3</a>]</li> <li>Developed UNILIPID, a method and specialized bioinformatic Python tool for adapting lipid–protein interaction parameters, enabling the extension of implicit membrane representations from all-atom to coarse-grained models, and allowing simulations with curved membrane geometries [<a href="Andr%C3%A9%20Lanrezac%20and%20Marc%20Baaden.%20%E2%80%9CUNILIPID,%20a%20Methodology%20for%20Energetically%20Accurate%20Prediction">~3</a>]</li> <li>Participated in the maintenance and release process of BioSpring by building a CI/CD pipeline (GitHub Actions, Docker) to automate releases, resolve multi-year dependency issues, and enhance software accessibility</li> </ul> <dl> <dt><strong>Bioinformatics Drug Design Intern</strong></dt> <dd><strong>Feb 2019 - June 2019</strong></dd> </dl> <p><a href="https://impmc.sorbonne-universite.fr/fr/equipes/biophysique_et_bioinformatique.html" rel="external nofollow noopener" target="_blank">Bioinformatics Research Team, ISYEB · Sorbonne université , Paris</a></p> <p>Supervised by <a href="http://www-ext.impmc.upmc.fr/~stratmann/" rel="external nofollow noopener" target="_blank">Dirk Stratmann</a></p> <ul> <li> <p>Developed a workflow combining Autodock with REMD simulations in GROMACS to assess binding affinity and stability of cyclic peptides, comparing results with HADDOCK to optimize peptide designs</p> </li> <li> <p>Manually designed cyclic peptides using UCSF-Chimera to optimize sequences and structures for targeted protein pockets, ensuring high binding affinity and stability</p> </li> </ul> <dl> <dt><strong>Bioinformatics Structural Analysis Intern</strong></dt> <dd><strong>July 2018</strong></dd> </dl> <p><a href="https://isyeb.mnhn.fr/fr/atelier-de-bio-informatique-384" rel="external nofollow noopener" target="_blank">Bioinformatics Research Team, ISYEB · Sorbonne université , Paris</a></p> <p>Supervised by <a href="https://isyeb.mnhn.fr/fr/annuaire/mathilde-carpentier-368" rel="external nofollow noopener" target="_blank">Mathilde Carpentier</a></p> <ul> <li> <p>Continued development of <a href="https://pubmed.ncbi.nlm.nih.gov/16049912/" rel="external nofollow noopener" target="_blank">YAKUSA</a>, a fast protein structural similarity search tool based on shared high-scoring structural pairs (SHSPs)</p> </li> <li> <p>Re-implemented Bellman’s algorithm (Python) to compute the longest path through shared motifs and optimize similarity search</p> </li> <li> <p>Developed heuristic algorithms to accelerate structural alignment and improve matching speed</p> </li> </ul> <h2 id="skills">Skills</h2> <p><strong>Programming:</strong> Python, C++, C#, Bash, C, HTML, CSS</p> <p><strong>Softwares &amp; Frameworks:</strong> Django, React, Unity, CMake, SQL</p> <p><strong>DevOps:</strong> Git, CI/CD (GitHub Actions), Docker</p> <p><strong>Languages:</strong> French (native), English (proficient), Mandarin (beginner)</p> <h2 id="publications">Publications</h2> <p>simulations”. WIREs Computational Molecular Science (July 2022). DOI:<a href="https://onlinelibrary.wiley.com/doi/10.1002/wcms.1594" rel="external nofollow noopener" target="_blank">10.1002/wcms.1594</a>.</p> <p>(Nov. 2022). DOI:<a href="https://www.mdpi.com/1999-4893/15/11/415" rel="external nofollow noopener" target="_blank">10.3390/a15110415</a>.</p> <p>of Protein Insertion into Implicit Membranes of Arbitrary Shape”. Membranes (Mar. 2023). DOI:<a href="https://www.mdpi.com/2077-0375/13/3/362" rel="external nofollow noopener" target="_blank">10.3390/membranes13030362</a>.</p> <p>Module in Chemistry, Molecular Sciences and Chemical Engineering. Elsevier, 2023. DOI:<a href="https://linkinghub.elsevier.com/retrieve/pii/B978012821978200115X" rel="external nofollow noopener" target="_blank">10.1016/B978-0-12-821978-2.00115-X</a>.</p> </body></html>